core/src/main/java/org/apache/accumulo/core/client/impl/ClientConfConverter.java

```diff
@@ -252,19 +251,10 @@ public static ClientConfiguration toClientConf(AccumuloConfiguration conf) {
       clientConf.setProperty(KERBEROS_SERVER_PRIMARY, krbName.getServiceName());
     }
 
-    HashSet<String> clientKeys = new HashSet<>();
-    for (ClientConfiguration.ClientProperty prop : ClientConfiguration.ClientProperty.values()) {
-      clientKeys.add(prop.getKey());
-    }
-
-    String key;
     for (Map.Entry<String,String> entry : conf) {
-      key = entry.getKey();
-      if (clientKeys.contains(key)) {
-        clientConf.setProperty(key, entry.getValue());
-      }
+      String key = entry.getKey();
+      clientConf.setProperty(key, entry.getValue());
```

Comment:
I actually think it's okay to preserve the code which filters out the invalid properties. This is a one-way conversion to what we need internally... it shouldn't contain anything that we don't recognize internally. If somehow keeping that stuff around  avoids a failure, we should not mask that failure by including those unknown properties.

Reply:
Ok I will remove this and limit the PR to the fix.

Reply:
See 23c3044db